### PR TITLE
Mo homescript

### PR DESCRIPTION
**Pull request by moffutt10**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,8 +1,57 @@
-console.log("I am linked");
+// Objects
+const vanGoghsEar = [{
+    name: "Van Gogh's Ear",
+    type: "Nashville Jazz Collective",
+    image: "https://onerecordperday.files.wordpress.com/2013/02/day48-plot.jpg",
+    biography:
+    `To describe Van Gogh’s Ear as simply a jazz group would be to commit 
+    an egregious disservice to what VGE actually is. Less a band and more 
+    an esoteric thinktank, VGE has found its niche within the vast and
```
Comment (Waysidetester) [inline main.js]: thinktank should be either think-tank or think tank

**Comment by colinlwhite** [comment]
💯

File: main.js (hunk 1 above)
Comment (Waysidetester) [reply main.js]: Everything else works

**Comment by Waysidetester** [comment]
👍

**Pull request merged**